2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,7 +18,17 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+    // write code here
+    const newObject = new Object() ;
+for (let countryCurrencyCode of countryCurrencyCodes){
+   
+        let countryCode = countryCurrencyCode[1];
+        let country = countryCurrencyCode[0];
```

Comment:
Again, here we can use const. It may seem that we are reassigning these variables as we loop through the array, but we aren't, we are creating new ones on each iteration. This means they are never reassigned and disappear after each loop iteration.